srcopsmetrics/create_bot_knowledge.py

```diff
@@ -29,15 +29,31 @@
 from github import Github, GithubObject, Issue, IssueComment, PullRequest, PullRequestReview, PaginatedList
 from github.Repository import Repository
 
+from thoth.storages.exceptions import NotFoundError
+from thoth.storages.ceph import CephStore
+
 _LOGGER = logging.getLogger(__name__)
 
 _GITHUB_ACCESS_TOKEN = os.getenv("GITHUB_ACCESS_TOKEN")
 
 ISSUE_KEYWORDS = {"close", "closes", "closed", "fix", "fixes", "fixed", "resolve", "resolves", "resolved"}
 
-
 STANDALONE_LABELS = {"size"}
 
+PREFIX = "data/thoth/dtuchyna/"
+HOST = "https://s3.upshift.redhat.com/"
+# env variables needed for ceph connection
+# THOTH_CEPH_KEY_ID =
+# THOTH_SECRET_KEY =
+BUCKET = "DH-DEV-DATA"
```

Comment:
They should be external variables provided. Take a look at https://github.com/thoth-station/graph-sync-job/blob/820ebc142f3042f10fe188ee2fca1490107f112f/openshift/syncJob-template.yaml#L145